cmd/release-controller/info.go

```diff
@@ -483,7 +485,7 @@ class FileServer(SimpleHTTPServer.SimpleHTTPRequestHandler):
             try:
                 self._present_default_content(name)
 
-                subprocess.check_output(["oc", "adm", "release", "extract", "--tools", "--to", name, "--command-os", "*", "registry.svc.ci.openshift.org/ocp/release:%s" % name],
+                subprocess.check_output(["oc", "adm", "release", "extract", "--tools", "--to", name, "--command-os", "*", "registry.svc.ci.openshift.org/RELEASE_NAMESPACE/release:%s" % name],
```

Comment:
Just make this an environment variable, and set that on the pod definition.  Don't use string replacement.